README.md

```diff
@@ -63,3 +63,17 @@ ssh root@rpi3
 # Enter password “raspberry”
 ```
 
+Note that the default firewall rules only allow SSH access from the local
+network. If you wish to enable SSH access globally, first change your root
+password using `passwd`. Next, issue the following commands to remove the
+corresponding firewall rules:
+
+```shell
+sudo iptables -D INPUT 6
+sudo ip6tables -D INPUT 4
```

Comment:
Specifying the rule numbers is fairly brittle. We’ll likely either forget to update this when changing the firewall rules, or — worse — people might put this into automation based on this image.

Let’s change this to `iptables -F INPUT` and `ip6tables -F INPUT` to flush the entire INPUT chain for each address family.

Also, I think we can drop the `sudo`, since working as root is implied throughout the README.